powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/provider/PowerAuthEncryptionProviderBase.java

```diff
@@ -165,7 +166,7 @@ public void decryptRequest(@Nonnull HttpServletRequest request, @Nonnull Type re
             final byte[] sharedInfo2Base = Base64.getDecoder().decode(encryptorParameters.sharedInfo2());
             final ServerEncryptor serverEncryptor = encryptorFactory.getServerEncryptor(
                     encryptorData.getEncryptorId(),
-                    new EncryptorParameters(version, applicationKey, activationId),
+                    new EncryptorParameters(version, applicationKey, activationId, null),// TODO Lubos - merge with Petr's branch
```

Comment:
Will be fix after merge of https://github.com/wultra/powerauth-restful-integration/pull/546